snap/snapcraft.yaml

```diff
@@ -9,17 +9,17 @@ apps:
     server:
         daemon: simple
         command: rel/couchdb/bin/snap_run
-        plugs: [network-bind, process-control, mount-observe, browser-support]
+        plugs: [network-bind, process-control, mount-observe]
     couchdb:
         command: rel/couchdb/bin/snap_run
-        plugs: [network-bind]
+        plugs: [network-bind,browser-support]
 parts:
     couchdb: 
         plugin: make
-        #source: http://www-us.apache.org/dist/couchdb/source/2.0.0/apache-couchdb-2.0.0.tar.gz
-        source: https://github.com/apache/couchdb.git
-        source-type: git
-        source-tag: 2.1.0
+        source: http://www-us.apache.org/dist/couchdb/source/2.2.0/apache-couchdb-2.2.0.tar.gz
+        #source: https://github.com/apache/couchdb.git
```

Comment:
Actually, GitHub is the master, surprisingly. You can leave this as it was.

BUT.

It would be best if you can switch to using our tarball releases instead - if you do this, you don't need a bunch of build dependencies (no node/npm, no sphinx) and the build goes a lot faster.

Tarball releases are here: https://www-us.apache.org/dist/couchdb/source/<release number>/

The advantage to switching to this download approach is that you can pull the GPG key and a SHA256/SHA512 to ensure you have the correct build. If you're curious how to do this, check out our [Dockerfile](https://github.com/apache/couchdb-docker/blob/master/2.2.0/Dockerfile#L126-L140) which has an example script doing precisely that.

Reply:
@sklassen If you'd like to explore this as an alternative, be my guest! We've put a lot of work into the packages, and if they can be used to drive a snap, that's better for everyone since we don't have to keep build options/etc synchronized between all things.